### DISCUSSION TITLE
Docker image?

### DISCUSSION THREAD

**Discussion by ronaldt80** [kalbasit/signal-api-receiver#43] [General]
Hi, really cool stuff - i was exploring the use of signal to tap into an external AI api but like you run json rpc for speed. Would be awesome to communicate with HA through signal.

There is reference to a docker file in your repo, where is the image located? Or is this Kubernetes only?

**Comment by kalbasit** [comment]
@ronaldt80 the images are on Dockerhub: https://hub.docker.com/r/kalbasit/signal-api-receiver I'm going to mention in the readme and improve the dockerhub page in #44.

I'm working on an aarch64 version in #41.

**Comment by ronaldt80** [comment]
Thanks! Will try today!
